applications/MedApplication/custom_io/med_model_part_io.cpp

```diff
@@ -371,7 +374,9 @@ class MedModelPartIO::MedFileHandler
             // basic checks if the file is compatible with the MED library
             med_bool hdf_ok;
             med_bool med_ok;
-            const med_err err = MEDfileCompatibility(rFileName.c_str(), &hdf_ok, &med_ok);
+
+            const med_err err = MEDfileCompatibility(med_file_mame.c_str(), &hdf_ok, &med_ok);
```

Comment:
maybe `rFileName.string().c_str()` would also work :man_shrugging:

Reply:
not it does not :D